Gemfile

```diff
@@ -12,6 +12,7 @@ gem 'turbolinks'
 gem 'config', '~> 1.0.0'
 gem 'redcarpet', '~> 3.3'
 gem 'friendly_id', '~> 5.1.0'
+gem 'devise', '3.4.1'
```

Comment:
It's good to see that this is pinned to a specific version, because I think we needed to do that with Krikri.